app/AccountService.js

```diff
@@ -0,0 +1,37 @@
+var playlistsArray = JSON.parse(usersJSON);
```

Comment:
Pay very close attention to how you name your variables. This name is obviously not ok and misleading. better allUsersList